.github/workflows/release.yml

```diff
@@ -0,0 +1,112 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - ci/*
```

Comment:
remove?

Reply:
The release, unlike docker publish, was triggered on pushes to ci/* branches in CircleCI (see https://github.com/filecoin-project/lotus/blob/78d9d9baa054448f2dfb192101eb9a9fbdd0ac9b/.circleci/config.yml#L1112).

All in all, releases were triggered on:
- push to ci/* (dry run)
- push to release/* (dry run)
- tag push

I think whether we perform the dry runs or not comes down to the team preference.

```suggestion
```

Reply:
Ah, I see. @arajasek Do we actually use this?

Reply:
Okay, actually this is kind of nice to have. But I'd like to find some way to avoid passing tokens because they're really powerful. I guess I can also just restrict pushes these branches, but in that case, I'd probably rename it something like test-release or something like that.

Reply:
I applied the suggestion from https://github.com/filecoin-project/lotus/pull/11785/files#r1542909114. That way, we'll only pass the tokens to goreleaser action if it's a run where publish=true.